src/views/schemaBrowser.ts

```diff
@@ -103,7 +104,197 @@ export default class schemaBrowser {
+
+          vscode.commands.executeCommand(`vscode-db2i.runEditorStatement`, {
+            content,
+            type: `statement`,
+            open: false,
+          });
+        }
+      }),
+
+      vscode.commands.registerCommand(`vscode-db2i.getIndexes`, async (object: SQLObject) => {
+        if (object) {
+          const content = `SELECT * FROM QSYS2.SYSINDEXSTAT WHERE TABLE_SCHEMA = '${object.schema}' and TABLE_NAME = '${object.name}'`;
+          vscode.commands.executeCommand(`vscode-db2i.runEditorStatement`, {
+            content,
+            type: `statement`,
+            open: false,
+          });
+        }
+      }),
+      
+      vscode.commands.registerCommand(`vscode-db2i.advisedIndexes`, async (object: SQLObject) => { //table
+        if (object) {
+          const content = `SELECT * FROM QSYS2.SYSIXADV WHERE TABLE_SCHEMA = '${object.schema}' and TABLE_NAME = '${object.name}' ORDER BY TIMES_ADVISED DESC`;
+          vscode.commands.executeCommand(`vscode-db2i.runEditorStatement`, {
+            content,
+            type: `statement`,
+            open: false,
+          });
+        }
+      }),
+
+      vscode.commands.registerCommand(`vscode-db2i.clearAdvisedIndexes`, async (object: SQLObject) => {
+        if (object) {
+          const result = await vscode.window.showWarningMessage(`Are you sure you want to clear all of the advised index rows from the Index Advisor for ${object.name}?`,  {
+            modal: true,
+          }, 'No', 'Yes');
+          
+          if(result === 'Yes') {
+            try {
+              await Table.clearAdvisedIndexes(object.schema, object.name);
+            } catch (e) {
+              vscode.window.showErrorMessage(e.message);
+            }
+          }
+        }
+      }),
+
+      vscode.commands.registerCommand(`vscode-db2i.deleteObject`, async (object: SQLObject) => {
+        if (object) {
+          const result = await vscode.window.showWarningMessage(`Are you sure you want to delete ${object.name}?`, {
+            modal: true,
+          }, 'No', 'Yes');
+
+          if(result === 'Yes') {
+            try {
+              await vscode.window.withProgress({
+                location: vscode.ProgressLocation.Notification,
+                title: `Deleting ${object.name}...`
+              }, async () => {
+                await Schemas.deleteObject(object.schema, object.name, object.type);
+              });
+
+              vscode.window.showInformationMessage(`${object.name} deleted`);
+              this.cache = {};
+              this.refresh();
+            } catch (e) {
+              vscode.window.showErrorMessage(e.message);
+            }
+          }
+        }
+      }),
+
+      vscode.commands.registerCommand(`vscode-db2i.renameObject`, async (object: SQLObject) => {
+        if (object) {
+          const name = await vscode.window.showInputBox({
+            title: "New File",
+            prompt: "Enter new name",
```

Comment:
Pre-populate the rename prompt with the current object name.
`value: object.name`